Assets/HoloToolkit/SpectatorView/Scripts/Networking/NewDeviceDiscovery.cs

```diff
@@ -38,7 +35,7 @@ public MarkerDetectionHololens MarkerDetectionHololens
 
         private void Awake()
         {
-#if NETFX_CORE
+#if WINDOWS_UWP
```

Comment:
I wonder what the difference is between `UNITY_WSA` and `WINDOWS_UWP`

Reply:
I've used `#WINDOWS_UWP` like `#UNITY_WSA && !UNITY_EDITOR`, useful when you're using platform, non-Unity APIs.

Reply:
So is it a logical replacement?

Should we update the usages of `#UNITY_WSA && !UNITY_EDITOR` in other places?

Reply:
We probably should, over time.

Reply:
Kev, could we add this to best practices and coding guidelines?
I like this table.